src/app/components/home-page/home-page.component.html

```diff
@@ -223,14 +223,35 @@ <h3 class="sub-title Previous">Previous Rockstars Speakers</h3>
   <div id="subscribe">
     <div class='row center-lg center-xs center-md middle-lg middle-md middle-sm
       middle-xs'>
-      <div class="col-xs-12 col-md-12" style='height: 150px'>
-        <div class="sub-title" style='line-height:150px;font-size: 2em'>
-          Get notified about important conference updates
+      <div class="col-xs-12 col-md-12" style='height: max-content;'>
+        <br>
+        <div class="sub-title" style='line-height:50px;font-size: 2em'>
+          Contact Us
+        </div>
+        <div class="sub-title" style='line-height:70px;font-size: 1.4em'>
+          gdggandhinagar@gmail.com
```

Comment:
Make it clickable.
and `href="mailto:$email"`

Reply:
$email? or email address